frontend/js/pages/BattleDetail.js

```diff
@@ -71,8 +75,8 @@ function BattleDetail(props) {
 }
 
 const mapStateToProps = (store) => ({
-  battle: store.battle,
-  user: store.user,
+  battle: store.battle.battle,
+  user: store.user.user,
```

Comment:
Is there a way to remove this nesting behavior? If is to difficult please use lodash: https://lodash.com/docs/4.17.15#get

Reply:
Check for changes on the reducers